keystone/server/flask/core.py

```diff
@@ -79,6 +79,14 @@
                       'config_file': '/etc/keystone/watcher.yaml',
                       'include_initiator_user_id_in_metric': 'true',
                       'include_target_project_id_in_metric': 'false'}),
+    # CCloud: add rate_limit middleware
+    _Middleware(namespace='rate_limit.middleware',
+                ep='rate-limit',
+                conf={'config_file': '/etc/keystone/ratelimit.yaml',
+                      'service_type': 'identity',
+                      'rate_limit_by': 'initiator_project_id',
+                      'backend_host': 'keystone-sapcc-rate-limit',
+                      'backend_timeout_seconds': '20'}),
```

Comment:
If you leave this at 20 seconds, in case rate-limit Redis is down, each request will get 20 seconds slower. 
Please set it to 1 second.